src/cljs/main/broadfcui/page/workspace/summary/library_view.cljs

```diff
@@ -31,7 +34,21 @@
            (when (:expanded? @state)
              [:div {}
               (map (partial library-utils/render-property library-schema library-attributes) (-> library-schema :display :secondary))
-              (library-utils/render-consent-codes library-schema library-attributes)])
+              (if orsp-id
+                (cond
+                  (:consent @state) (library-utils/render-consent orsp-id (:consent @state))
+                  (:consent-error @state) (library-utils/render-consent-error orsp-id (:consent-error @state))
+                  :else (library-utils/render-library-row (str "Retrieving information for " orsp-id) [comps/Spinner]))
+                (library-utils/render-consent-codes library-schema library-attributes))])
            [:div {:style {:marginTop "0.5em"}}
             (style/create-link {:text (if (:expanded? @state) "Collapse" "See more attributes")
-                                :onClick #(swap! state update :expanded? not)})]]])]))})
+                                :onClick #(swap! state update :expanded? not)})]]])]))
+   :component-did-mount
+   (fn [{:keys [props state]}]
+     (when-let [orsp-id (:library:orsp (:library-attributes props))]
+       (endpoints/get-consent orsp-id
+         (fn [{:keys [success? get-parsed-response]}]
+           (let [response (get-parsed-response)]
+             (if success?
+               (swap! state assoc :consent (get-parsed-response))
+               (swap! state assoc :consent-error (get-parsed-response))))))))})
```

Comment:
Thank you - I was using `let` for debugging the response which is no longer needed!